Drivers/IWDG_driver/Inc/IndependentWatchdog.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * IndependentWatchdog.h
+ *
+ *  Created on: Jun. 22, 2023
+ *      Author: pratyushmakkar
+ */
+
+#ifndef SRC_DRIVERS_INDEPENDENTWATCHDOG_H_
+#define SRC_DRIVERS_INDEPENDENTWATCHDOG_H_
+
+
+#include <stdint.h>
+#include "main.h"
+
+class IWDG_t {
+	private:
+		uint8_t iwdg_prescaler;
+		uint16_t iwdg_reload;
```

Comment:
I think we would be better off storing a `IWDG_HandleTypeDef` instead of these members.

Reply:
I have modified it to IWDG_HandleTypeDef* in the latest commit.